### PR TITLE
Create new disambig dag

### PR DESCRIPTION
**Pull request by bethanne-card**

### CODE REVIEW

File: airflow/dags/disambiguation/inventor_assignee_disambiguation.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class SQLTemplatedPythonOperator(PythonOperator):
 disambiguation = DAG(
     dag_id='inventor_assignee_disambiguation',
     default_args=default_args,
-    description='Perform inventor & assignee disambiguation',
+    description='Perform inventor, assignee, & location disambiguation',
```
Comment (sarvothaman) [inline airflow/dags/disambiguation/inventor_assignee_disambiguation.py:60]: Are we doing anything location related in this dag? We should probably clean those up and remove this description if not

File: airflow/dags/disambiguation/inventor_assignee_disambiguation.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
     finalize_disambiguation, upload_results as upload_inventor_results
 from updater.disambiguation.location_disambiguation.location_disambiguator import *
 from updater.post_processing.post_process_location import post_process_location, post_process_qc
-from updater.post_processing.post_process_assignee import post_process_qc as qc_post_process_assignee, \
+from updater.post_processing.post_process_assignee import additional_post_processing_assignee, post_process_qc as qc_post_process_assignee,  \
```
Comment (sarvothaman) [inline airflow/dags/disambiguation/inventor_assignee_disambiguation.py:21]: I see the additional post processing is set to run after canonical gets created. I suppose the canonical gets recreated within additional post processing step?

File: updater/post_processing/post_process_assignee.py [hunk 3]
```diff
@@ -188,19 +200,178 @@ def precache_assignees(**kwargs):
 
 
 def create_canonical_assignees(**kwargs):
-    config = get_current_config(schedule='quarterly',**kwargs)
+    config = get_current_config(schedule='quarterly', **kwargs)
```
Comment (sarvothaman) [inline updater/post_processing/post_process_assignee.py:203]: Should this function also create the view `assignee` to link it to `assignee_YYYYMMDD`?

**Review by sarvothaman — APPROVED** [review]
Asked a few questions, not a lot of changes.

File: airflow/dags/disambiguation/inventor_assignee_disambiguation.py (hunk 2 above)
Comment (bethanne-card) [reply airflow/dags/disambiguation/inventor_assignee_disambiguation.py:21]: yes precache and create_canonical_assignees are regenerated in that step

File: airflow/dags/disambiguation/inventor_assignee_disambiguation.py (hunk 1 above)
Comment (bethanne-card) [reply airflow/dags/disambiguation/inventor_assignee_disambiguation.py:60]: Yes we are still creating the canonical_locations table quarterly

File: updater/post_processing/post_process_assignee.py (hunk 3 above)
Comment (bethanne-card) [reply updater/post_processing/post_process_assignee.py:203]: Yup it does: 
    target_table = "assignee_{suffix}".format(suffix=suffix)
    create_sql = """
    CREATE TABLE `{target_table}` (
    `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
  `type` int(4) DEFAULT NULL,
  `name_first` varchar(96) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `name_last` varchar(96) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `organization` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `version_indicator` date NOT NULL DEFAULT '{version_indicator}',
  `created_date` timestamp NOT NULL DEFAULT current_timestamp(),
  `updated_date` timestamp NULL DEFAULT NULL ON UPDATE current_timestamp(),
  PRIMARY KEY (`id`),
  KEY `assignee_version_indicator_index` (`version_indicator`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
    """.format(version_indicator=version_indicator, target_table=target_table)
    view_query = """
    CREATE OR REPLACE VIEW assignee as select * from {target_table}
    """.format(target_table=target_table)

**Pull request merged**